test/e2e/nodemaintenance_test.go

```diff
@@ -282,6 +306,7 @@ func nodeMaintenanceTest(t *testing.T, f *framework.Framework, ctx *framework.Te
 		t.Fatal(err)
 	}
 
+	checkOperatorCRD(t)
```

Comment:
I'd love to have this as a functional test rather than e2e, but it doesn't seem possible